### PR TITLE
Close #3097, using with WebStorm.

### LINKED ISSUE #3097: Editor settings for WebStorm
Something for the editors section of the ember-cli website.

In order to configure WebStorm for use with ember-cli, see below.

`File > Settings > Languages & Frameworks > JavaScript > ECMAScript6`

Right mouse on the following directories and mark accordingly.

Mark as `excluded`:
* /tmp

Mark as `resource route`:
* /
* /bower_components
* /bower_components/ember-qunit/lib
* /public

Mark as `test sources root`:
* /tests

Add the following line to `.gitignore`
```
.idea
```

### PR DESCRIPTION
**Pull request by jluckyiv**
Add instructions for use with WebStorm per #3097.

### CODE REVIEW

File: _posts/2013-04-02-editors.md [hunk 1]
```diff
@@ -27,3 +27,35 @@ If you are using [SublimeText 3](http://www.sublimetext.com) with `ember-cli`, b
 // project basis.
 "folder_exclude_patterns": [".svn", ".git", ".hg", "CVS", "tmp/class-*", "tmp/es_*", "tmp/jshinter*", "tmp/replace_*", "tmp/static_compiler*", "tmp/template_compiler*", "tmp/tree_merger*", "tmp/coffee_script*", "tmp/concat-tmp*", "tmp/export_tree*", "tmp/sass_compiler*"]
 {% endhighlight %}
+
+### WebStorm
+If you are using [WebStorm](https://www.jetbrains.com/webstorm/) with `ember-cli`, you will need to modify your `.gitignore` file and `ECMAScript6` settings.
+
+First, add the following line to `.gitignore`:
+```
```
Comment (twokul) [inline _posts/2013-04-02-editors.md]: you probably should use `{% highlight %} {% endhighlight %}`

File: _posts/2013-04-02-editors.md [hunk 2]
```diff
@@ -27,3 +27,35 @@ If you are using [SublimeText 3](http://www.sublimetext.com) with `ember-cli`, b
...
+
+Next, from the WebStorm menu:
+
+`File > Settings -> Languages & Frameworks -> JavaScript -> ECMAScript6`
+
+Right-click on each of the following directories and mark as indicated:
+
+Mark as `excluded`:
+```
```
Comment (twokul) [inline _posts/2013-04-02-editors.md]: same here

File: _posts/2013-04-02-editors.md (hunk 1 above)
Comment (jluckyiv) [inline _posts/2013-04-02-editors.md]: @twokul Thanks for the comments. Sorry for being thick, but should the "highlight" tags surround the text on line 34 only? Or 32 through 34?

File: _posts/2013-04-02-editors.md (hunk 2 above)
Comment (jluckyiv) [inline _posts/2013-04-02-editors.md]: @twokul Thanks again, but which lines? I'm sure I'm missing some kind of marking.

File: _posts/2013-04-02-editors.md (hunk 1 above)
Comment (kgish) [inline _posts/2013-04-02-editors.md]: Would recommend changing:

If you are using [WebStorm](https://www.jetbrains.com/webstorm/) with `ember-cli`, you will need to modify your `.gitignore` file and `ECMAScript6` settings.

to

If you are using [WebStorm](https://www.jetbrains.com/webstorm/) with `ember-cli`, you will need to modify your `.gitignore` file, enable `ECMAScript6` settings and mark certain directories.

File: _posts/2013-04-02-editors.md [hunk 3]
```diff
@@ -27,3 +27,35 @@ If you are using [SublimeText 3](http://www.sublimetext.com) with `ember-cli`, b
...
+.idea
+```
+
+Next, from the WebStorm menu:
+
+`File > Settings -> Languages & Frameworks -> JavaScript -> ECMAScript6`
+
+Right-click on each of the following directories and mark as indicated:
+
+Mark as `excluded`:
```
Comment (Kuzirashi) [inline _posts/2013-04-02-editors.md]: I think `/dist` should be also marked as excluded.

Comment (stefanpenner) [inline _posts/2013-04-02-editors.md]: :+1:

File: _posts/2013-04-02-editors.md (hunk 1 above)
Comment (twokul) [inline _posts/2013-04-02-editors.md]: @jluckyiv basically replace "code" ticks with `highlight/endhighlight` and it's all :+1:

**Comment by jluckyiv** [comment]
@twokul I replaced the backticks with {% highlight %}. @Kuzirashi I added /dist to excluded. @kgish I made the changes you suggested.

**Comment by stefanpenner** [comment]
> Should I squash the commits and push?

only if you want

**Pull request merged**